scripts/vfx/build_otio.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Copyright (c) Contributors to the aswf-docker Project. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+git clone https://github.com/PixarAnimationStudios/OpenTimelineIO.git
```

Comment:
Would it make sense to have a centralized file that has the canonical repo locations for all the projects instead of having those in the project-specific build scripts? At some point the OTIO repo will move to the AcademySoftwareFoundation GitHub organization, and since builds will still work due to GitHub redirects, it could be easy to forget to update this hard coded repo path.

Reply:
I think I will replace these scripts by `rez` packages pretty soon, and at that point we could indeed centralise so repository locations, at least for the ASWF repos... What do you think?

Reply:
The OTIO team will enjoy this :) Thanks so much Aloys!!

Reply:
Rez packages are an intriguing notion! I'm sure several of us can keep an eye open to have this script updated when the repo migrates to ASWF.

Reply:
I don't know enough about Rez to comment on the implications, but sounds like you have a plan!